multiplex_immuno_processing/execute_single_position_alignment.py

```diff
@@ -0,0 +1,191 @@
+import argparse
```

Comment:
no need to name this file `single_position`. It can just take an argument to do single position and keep the same name as before.

Reply:
This is actually the script that the jinja tempelate calls and its by default set to only do alignment for a single position. But yeah, the naming convention is weird--- will change that

Reply:
sounds good

Reply:
I decided to keep the single position naming convention here because this is only doing it on a single position w/ the jinja tempelate, but we can change this later